src/components/Delegation/dashboard/DelegatedStakepoolInfo.js

```diff
@@ -12,6 +12,7 @@ import {
   Animated,
 } from 'react-native'
 import {injectIntl, defineMessages, intlShape} from 'react-intl'
+import _ from 'lodash'
```

Comment:
prefer to import specific functions when possible to make better use of tree shaking